sandbox/cgroup/cgroup_test.go

```diff
@@ -236,6 +236,24 @@ func (s *cgroupSuite) TestProgGroupBadSelector(c *C) {
 	c.Check(group, Equals, "")
 }
 
+func (s *cgroupSuite) TestParsePid(c *C) {
+	pid, err := cgroup.ParsePid("10")
```

Comment:
You could probably test parsePids instead. It's used in PidsInGroup and calls to parsePid internally.

Reply:
I think this is fine as is. Thinking about it, perhaps this should move to osutil instead?